docs/reference/contributing/target/QuadSPI.md

```diff
@@ -1,25 +1,50 @@
 <h3 id="quadspi-port">QuadSPI</h3>
 
-[Include a brief description here.]
+The QSPI HAL defines API for targets that contain QSPI capable peripheral. Developers often use the QSPI interface for data storage.
```

Comment:
The QSPI HAL defines API for targets? What does this mean? Please clarify.

Reply:
Fixed